package.json

```diff
@@ -61,6 +61,7 @@
     "rimraf": "^2.4.0",
     "style-loader": "^0.12.4",
     "url-loader": "~0.5.5",
+    "victory-util": "^1.0.0",
```

Comment:
Will actually depend on `1.1.0` which doesn't exist yet – `npm link` the victory-util branch to test out the demo.